Formula/cctools.rb

```diff
@@ -47,10 +47,16 @@ class Cctools < Formula
       sha256 "f49162b5c5d2753cf19923ff09e90949f01379f8de5604e86c59f67441a1214c"
     end
 
-    # Fix building libtool with LTO disabled
-    patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/db27850/cctools/libtool-no-lto.diff"
-      sha256 "3b687f2b9388ac6c4acac2b7ba28d9fd07f2a16e7d2dad09aa2255d98ec1632b"
+    # Fix building strings with LTO disabled
+    patch :p0 do
+      url "https://gist.githubusercontent.com/al3xtjames/8d68ef87c4c2138ef89d3d6e20879401/raw/d178e955c0f6cfcb91d775c016cb5ad95da43db0/cctools-895-strings-no-lto.diff"
+      sha256 "885302f6573d7bf472f7bff4600b6387234df9aea8cd4187e736e0fe89551eab"
+    end
+
+    # Fix objdump/llvm-objdump paths in otool
+    patch :p0 do
+      url "https://gist.githubusercontent.com/al3xtjames/ac4096fa2508af7024ba410fef67b9df/raw/5d601f9508e39249220ab7cd2b4fb310087c4969/cctools-895-otool-objdump-path.diff"
```

Comment:
How about setting this to a placeholder string, like `@@CLT_BINDIR@@`, that you can then `inreplace` within the formula? e.g.

```ruby
inreplace "otool/main.c", "@@CLT_BINDIR@@", "#{MacOS.active_developer_dir}/usr/bin"
```